nox/sessions.py

```diff
@@ -351,7 +355,11 @@ def imperfect(self):
         """
         if self.status == Status.SUCCESS:
             return "was successful"
-        return self.status.name.lower()
+        status = self.status.name.lower()
+        if self.reason:
```

Comment:
![nunchucks-nuns-chuck-norris-nun-12593488599](https://user-images.githubusercontent.com/520669/45960283-bffe0a80-bfd0-11e8-967b-6ab3c31c2c20.jpg)

Reply:
Intentional, it can be an empty string.